tasks/configuration/Linux.yml

```diff
@@ -69,15 +71,16 @@
     body: '{"set-user": {"{{ solr_auth_user }}" : "{{ solr_auth_pass }}"}}'
   become: True
   when:
+    - solr_auth_configure
     - solr_auth_type is defined
```

Comment:
Should we remove it?
`solr_auth_type is defined` in defaults so it be always True